TRANSLATIONS.rst

```diff
@@ -0,0 +1,145 @@
+the editor is English and you may select another available language
+using the Globe icon in the main menu.
+
+1.1 Crowdin
++++++++++++
+The Micro:bit Educational Foundation use the `Crowdin
+platform <https://crowdin.com/project/microbitorg>`__ to manage
+translations to manage and support community translations for it’s web
+projects. We have an amazing team of volunteer translators and
+proofreaders in our global micro:bit community and Crowdin helps us
+support them through the process. There is some more information on how
+this works at
+`microbit.org/translate <https://microbit.org/translate/>`__
+
+1.2 Translating the editor
+++++++++++++++++++++++++++
+We encourage translators to use
+`Crowdin <https://crowdin.com/project/microbitorg>`__ as it has a
+rigorous process for translation, proofreading and syncronisation and
+the Foundation use it to communicate and help guide the translation
+community.
+
+In the microbit.org project, you’ll find a `folder titled apps that
+contains a sub folder for the Python
+Editor. <https://crowdin.com/project/microbitorg/ar#/new/apps/python-editor>`__
+This folder contains the current English language translation
+**en.json** file with the strings required for translation. Selecting
+this file opens the Crowdin editor where you can submit your own or use
+a suggested translation for the string. It will then need to be verified
+by a proofreader for that language.
+
+1.3 Syncing Translations
+++++++++++++++++++++++++
+Crowdin can perform a sync between itself and Github, meaning that we can pass language updates either way. This
+is currently triggered manually. When a new release of the editor is
+made, any updates to the strings in en.js will be uploaded to Crowdin
+and translators are notified that translation strings have changed.
+
+**1.4 Requesting a language** There are two cases in which you may want
+to request a language: a) If you do not see the language available in
+the editor, but it is enabled in the `Crowdin
+project <https://crowdin.com/project/microbitorg>`__ b) If you do not
+see the language available in the editor or the Crowdin project In
+either case please `get in touch with the
+Foundation <https://support.microbit.org/en/support/tickets/new>`__ to
+ask if it can be made available.
+
+1.5 Available languages
++++++++++++++++++++++++
+The Foundation strives to make it’s online
+offerings as accessible as possible and new translations to the Crowdin
+project are always welcome. To ensure a consistent experience for users,
+we need to ensure that any languages we make available for translation
+are suitably maintained. Enabled languages can be found in the Crowdin
+project. For the Python Editor, we expect the **source strings to be
+100% translated and proofread** before they are made public as a
+released language.
+
+2. Release cycle
+----------------
+
+2.1 Releases and freezes
+++++++++++++++++++++++++
```

Comment:
I think we should simplify this to say that the string freeze is at done at the RC phase.